--- USD/cmake/defaults/msvcdefaults.cmake	Thu Mar 30 11:39:39 2017
+++ /c/temp/msvcdefaults.cmake	Thu Mar 30 11:39:29 2017
@@ -85,7 +85,7 @@
 
 # Forces all libraries that have separate source to be linked as
 # DLL's rather than static libraries on Microsoft Windows.
-_add_define("BOOST_ALL_DYN_LINK")
+# _add_define("BOOST_ALL_DYN_LINK")
 
 # Need half::_toFloat and half::_eLut.
 _add_define("OPENEXR_DLL")
@@ -104,3 +104,6 @@
 # Enable multiprocessor builds.
 set(_PXR_CXX_FLAGS "${_PXR_CXX_FLAGS} /MP")
 set(_PXR_CXX_FLAGS "${_PXR_CXX_FLAGS} /Gm-")
+
+# Define the "and" keyword for msvc
+#_add_define("and=&&")
