--- CMakeLists.txt	Wed Mar 22 09:19:25 2017
+++ /c/temp/CMakeLists.txt	Thu Mar 23 11:38:14 2017
@@ -98,16 +98,18 @@
 # but some linkers by default will error out when building this due
 # to the references to undefined symbols. So we need to specify a
 # link flag to force these symbols to be looked up at runtime.
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    set_target_properties(TestPlugDsoUnloadable
-        PROPERTIES
-        LINK_FLAGS "-undefined dynamic_lookup"
-    )
-elseif (CMAKE_SYSTEM_NAME STREQUAL "Windows")
-    set_target_properties(TestPlugDsoUnloadable
-        PROPERTIES
-        LINK_FLAGS "/FORCE:UNRESOLVED"
-    )
+if (${PXR_BUILD_TESTS})
+    if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+        set_target_properties(TestPlugDsoUnloadable
+            PROPERTIES
+            LINK_FLAGS "-undefined dynamic_lookup"
+        )
+    elseif (CMAKE_SYSTEM_NAME STREQUAL "Windows")
+        set_target_properties(TestPlugDsoUnloadable
+            PROPERTIES
+            LINK_FLAGS "/FORCE:UNRESOLVED"
+        )
+    endif()
 endif()
 
 pxr_create_test_module(TestPlugModule1
